independent-publisher-connectors/Proposal - RegEx Matching (Independent Publisher)/readme.md

```diff
@@ -0,0 +1,25 @@
+## RegEx Matching
+This connector extends Power Automate capabilities for performing tests whether a text string matches a pattern. There are pre-built patterns and also an option for user to provide their own regular expressions (regex) for matching the text string. 
+
+## Publisher
```

Comment:
@happyume please format the publisher, it should look like `Publisher: Publisher's Name`.

Reply:
This is now updated.